python/pickle/PyROOT_readtests.py

```diff
@@ -5,7 +5,7 @@
 
 """Pickle writing unit tests for PyROOT package."""
 
-import os, sys, unittest
+import os, sys, unittest, ctypes
```

Comment:
General comment: the modifications to the tests need to be executed only when we are in experimental PyROOT, the old PyROOT test code should remain the same. You need to follow a similar strategy as in the previous test you worked on, where you exported an environment variable to the test and, depending on the value of that variable, you had one behavior or another.